packages/components/CHANGELOG.md

```diff
@@ -16,7 +16,8 @@
 
 ### Internal
 
--   Renamed `PolymorphicComponent*` types to `WordPressComponent*` ([#34330](https://github.com/WordPress/gutenberg/pull/34330))
+-   Renamed `PolymorphicComponent*` types to `WordPressComponent*` ([#34330](https://github.com/WordPress/gutenberg/pull/34330)).
+-   Deleted the `createComponent` utility function ([#34929](https://github.com/WordPress/gutenberg/pull/34929)).
```

Comment:
This should be under Unreleased, no? I don't suppose it's going to trip anyone up though. I happened to notice this when I went to make a changelog edit in #35020 and we could update it in that PR if you'd like.

Reply:
You're correct — I missed that `Unreleased` header. Well spotted!

>  I went to make a changelog edit in #35020 and we could update it in that PR if you'd like

That would be great, thank you